### ISSUE TITLE
MultiSelect: Unable to select remove icon of chips through keyboard

### ISSUE THREAD

**Issue by RashmiChandran** [primefaces/primereact#7349]
### Describe the bug

To remove the chip, user needs to deselect the options through keyboard. 
https://github.com/user-attachments/assets/0bff7489-e26b-4a49-ac3f-9e83e488d837

### System Information

```Shell
System:
    OS: Windows 11 10.0.22631
    CPU: (12) x64 13th Gen Intel(R) Core(TM) i7-1355U
    Memory: 13.37 GB / 31.69 GB
  Binaries:
    Node: 20.10.0 - C:\Program Files\nodejs\node.EXE
    npm: 10.5.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Chromium (127.0.2651.74)
    Internet Explorer: 11.0.22621.3527
```

### Expected behavior

Remove icon of chips can be focused while tabbing through multiselect
